### DISCUSSION TITLE
v0.3.27

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#398] [Announcements]
## What's Changed
* Remove Reduce Spell Effect by @littlesera in https://github.com/littlesera/LSCG/pull/397

**Full Changelog**: https://github.com/littlesera/LSCG/compare/v0.3.26...v0.3.27

<hr /><em>This discussion was created from the release <a href='https://github.com/littlesera/LSCG/releases/tag/v0.3.27'>v0.3.27</a>.</em>

**Comment by QueenAsharah** [comment]
way to ruin my persona.... remove the one magic spell based around my character.... what a joke you are... abuse? wow.... what a load a shit whore

**Reply by jimmyG1** [reply]
yea really if you remove move one might as well remove both

**Reply by littlesera** [reply]
Sorry for the lack of transparency; Reduce was removed by request from Bondage Club administration.

**Reply by Shiranui-Izayoi** [reply]
What is the solution for? And how does whatever the issue is effect reduce but not enlarge I am genuinely curious.

Is there some issue with the game rendering or is it something else? Would just like to know what the issue is.

**Reply by littlesera** [reply]
![image](https://github.com/littlesera/LSCG/assets/126942824/b22c0a2e-6e53-418b-8c4b-a3974607edea)

**Reply by Shiranui-Izayoi** [reply]
Ahhh it is that... I guess I can somewhat see where he is coming from even if I don't agree with it. 

Since at least to me I would find it more telling/worrying if someones first thought upon seeing a fictional short and flat chested character is "this is a child". So I would personally think it is fine.

But as said I can see Ben's logic from a standpoint of just wanting to be really careful. Considering there are a lot of dumb people online. For proof I refer to Twitter/X.
